libvast/src/system/start_command.cpp

```diff
@@ -43,6 +43,10 @@ caf::message start_command_impl(start_command_extra_steps extra_steps,
                                 const command::invocation& invocation,
                                 caf::actor_system& sys) {
   VAST_TRACE(invocation);
+  // Bail out early for bogus invocations.
+  if (caf::get_or<bool>(invocation.options, "system.node", false))
+    return caf::make_message(make_error(ec::parse_error, "cannot start a local "
+                                                         "node"));
```

Comment:
```suggestion
    return caf::make_message(make_error(ec::invalid_configuration, "cannot start a local node"));
```

This needs text reflow as well.

Reply:
This was `clang-format`. I cannot commit this if I apply this change.

Reply:
I meant that you needed to do this change locally and then reformat for the text reflow. I've changed the error category.